### PR TITLE
Remove location parameter from v1

### PR DESCRIPTION
**Pull request by oyamauchi**
## Description

This is the final step of migrating to separate top-level `zip` and
`address` parameters: removing the old nested parameter.

The embed frontend is already switched over, and there should be no
other clients of v1, so although this is a breaking change it should
be safe.

## Test Plan

`yarn lint`, `yarn test`. Run the update-fixtures script and make sure
the updates all succeed and there are no changes.

### CODE REVIEW

**Comment by veekas** [comment]
Looks like Oregon-specific tests are failing

File: src/schemas/v1/utilities-endpoint.ts [hunk 1]
```diff
@@ -58,10 +53,10 @@ export const API_UTILITIES_SCHEMA = {
       },
     },
     oneOf: [
-      { required: ['location'] },
       { required: ['zip'] },
       { required: ['address'] },
     ],
+    additionalProperties: false,
```
Comment (veekas) [inline src/schemas/v1/utilities-endpoint.ts:59]: I'm assuming this is just to have some additional security?

File: yarn.lock [hunk 2]
```diff
@@ -1629,11 +1629,40 @@ doctrine@^3.0.0:
   dependencies:
     esutils "^2.0.2"
 
+dom-serializer@^2.0.0:
```
Comment (veekas) [inline yarn.lock:1632]: oh nice, thanks for checking in this yarn lock update

**Review by veekas — APPROVED** [review]

**Comment by oyamauchi** [comment]
BTW I checked the Cloud Run logs to make sure we completely stopped seeing this parameter when I merged rewiringamerica/embed.rewiringamerica.org#144

**Pull request merged**